sqlconnect/internal/redshift/driver/dsn.go

```diff
@@ -225,6 +283,13 @@ func (cfg *RedshiftConfig) GetRetryMaxAttempts() int {
 	return cfg.RetryMaxAttempts
 }
 
+func (cfg *RedshiftConfig) GetRoleARNExpiry() time.Duration {
+	if cfg.RoleARNExpiry <= 0 {
+		return 15 * time.Minute
+	}
+	return cfg.RoleARNExpiry
+}
+
```

Comment:
Since default value handling already happens in `AssumeRoleOptions` itself, we shouldn't need this method anymore.
```suggestion
```

Reply:
Yeah. Missed removing it.